objects.proto

```diff
@@ -6,22 +6,108 @@ option java_multiple_files = false;
 option java_package = "EcdarProtoBuf";
 option java_outer_classname = "ObjectProtos";
 
-message Zone {
-  int32 dimensions = 1;
-  repeated int32 matrix = 2;
-  // Maybe include a lookup table for the indices?
+import "component.proto";
+
+message ComponentInstance {
+  string component_name = 1;
+  uint32 component_index = 2;
+}
+
+message Clock {
+  message ComponentClock {
+    ComponentInstance component_instance = 1;
+    string clock_name = 2;
+  }
+  message SystemClock {
+    uint32 clock_index = 1;
+  }
+  message Zero {}
```

Comment:
Should we rename this to `ZeroClock` for consistency? Its purpose is pretty obvious in the way it is used, but maybe it would be nice to make it explicit

Reply:
I agree, it has been changed now :)